### PR TITLE
Fix for r2.0 16 12

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/media/src/lib/components/viewers/video-viewer/video-viewer.component.ts [hunk 1]
```diff
@@ -118,9 +118,21 @@ export class VideoViewerComponent implements AfterViewInit {
   toggleFullScreen() {
 
     if (!this.fullScreen) {
-      this.playerContainer.nativeElement.requestFullscreen();
+      if (!!this.playerContainer.nativeElement.requestFullscreen) {
```
Comment (GrandSchtroumpf) [inline libs/media/src/lib/components/viewers/video-viewer/video-viewer.component.ts:121]: @pldespaigne Could we create a reusable function to for fullscreen to not repeat ourselves ?
```typescript
function toggleFullscreen(element: HTMLElement) {
  // I didn't check the API in details
  const isFullscreen = (document.fullscreenElement || document.webkitFullscreenElement || document.mozFullScreenElement);
  if (isFullscreen) {
    element.requestFullscreen();
    element.webkitRequestFullscreen();
  } else {
    ...
  }
}
```

File: libs/ui/src/lib/tunnel/exit/exit.component.ts [hunk 2]
```diff
@@ -1,10 +1,8 @@
 import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
-import { TunnelService } from '../tunnel.service';
 import { Router, ActivatedRoute } from '@angular/router';
-import { boolean } from '@blockframes/utils/decorators/decorators';
 
 @Component({
-  selector: 'tunnel-exit',
+  selector: '[exitRedirect] tunnel-exit',
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/tunnel/exit/exit.component.ts:5]: This is not mandatory as you've got `this.exitRedirect || '/c/o/'`

File: libs/ui/src/lib/tunnel/exit/exit.component.ts [hunk 3]
```diff
@@ -14,19 +12,16 @@ export class ExitComponent implements OnInit {
...
-
   constructor(
-    private service: TunnelService,
     private router: Router,
     private route: ActivatedRoute
   ) { }
 
   ngOnInit() {
-    this.routeBeforeTunnel = this.service.previousUrl || this.exitRedirect || '/c/o/';
+    this.routeBeforeTunnel = this.exitRedirect || '/c/o/';
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/tunnel/exit/exit.component.ts:21]: This should be a getter incase the existRedirect change

File: apps/festival/festival/src/app/marketplace/event/view/view.component.html [hunk 4]
```diff
@@ -21,8 +21,16 @@
...
-          <div fxLayout fxLayoutAlign="center center" fxLayoutGap="16px">
-            <org-card fxFlex="400px" [org]="event.org"></org-card>
+          <div fxLayout fxLayoutAlign="start center" fxLayoutGap="16px">
+            <a matRipple [routerLink]="['/c/o/marketplace/organization', event.org.id]">
+              <img [ref]="event.org.logo" asset="empty_organization.webp" type="logo" alt="Organization Logo">
+            </a>
+            <div fxLayout="column" fxLayoutGap="4px">
+              <h6><a [routerLink]="['/c/o/marketplace/organization', event.org.id]">{{ event.org | orgName }}</a></h6>
+              <ng-container *ngIf="event.organizedBy as user">
+                <span>{{ user | displayName }}{{ user.position ? ' | ' + user.position : '' }}</span>
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/marketplace/event/view/view.component.html:31]: Create a pipe to manage this kind of logic. Don't overload the html with this kind of logic.

**Review by GrandSchtroumpf — CHANGES_REQUESTED** [review]

File: apps/festival/festival/src/app/marketplace/event/view/view.component.html (hunk 4 above)
Comment (phpgeekfr) [reply apps/festival/festival/src/app/marketplace/event/view/view.component.html:31]: I'm on this

Comment (phpgeekfr) [reply apps/festival/festival/src/app/marketplace/event/view/view.component.html:31]: fixed in https://github.com/blockframes/blockframes/pull/4574

File: libs/media/src/lib/components/viewers/video-viewer/video-viewer.component.ts (hunk 1 above)
Comment (phpgeekfr) [reply libs/media/src/lib/components/viewers/video-viewer/video-viewer.component.ts:121]: I take this

File: libs/ui/src/lib/tunnel/exit/exit.component.ts (hunk 2 above)
Comment (fritzschoff) [reply libs/ui/src/lib/tunnel/exit/exit.component.ts:5]: this is mandatory cause when you create a new movie and you exit it you would come to the lobby page and not in the movie view

File: libs/ui/src/lib/tunnel/exit/exit.component.ts (hunk 3 above)
Comment (fritzschoff) [reply libs/ui/src/lib/tunnel/exit/exit.component.ts:21]: its not going to change, you would need exit the tunnel comp

File: libs/media/src/lib/components/viewers/video-viewer/video-viewer.component.ts (hunk 1 above)
Comment (phpgeekfr) [reply libs/media/src/lib/components/viewers/video-viewer/video-viewer.component.ts]: factorization done in https://github.com/blockframes/blockframes/pull/4560/commits/0faecece9c0c75e6a74fea1746a62e941eab3c24

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**